tests/manager/models/conftest.py

```diff
@@ -0,0 +1,65 @@
+# conftest.py
+from uuid import UUID
+
+import pytest
+from graphene import Schema
+from graphene.test import Client
+
+from ai.backend.manager.models.gql import GraphQueryContext, Mutations, Queries
+
+
+@pytest.fixture(scope="module", autouse=True)
+def client() -> Client:
+    return Client(Schema(query=Queries, mutation=Mutations, auto_camelcase=False))
+
+
+@pytest.fixture(scope="module")
```

Comment:
Maybe you want it to have different fixture per each **function**?
Ref: https://pytest-with-eric.com/fixtures/pytest-fixture-scope/#Pytest-Fixture-Scope-%E2%80%94-Function
```suggestion
@pytest.fixture(scope="function")
```

Reply:
I implemented it this way assuming that the same context would be used on a per-file basis, but it seems that each function within the file may require a different context.